x-pack/platform/plugins/shared/features/server/feature_registry.ts

```diff
@@ -123,6 +123,12 @@ export class FeatureRegistry {
         feature.name = featureOverride.name;
       }
 
+      // We are allowing null values for description to be able to remove the description from the feature.
+      // The override is used in serverless.security.search_ai_lake.yml
```

Comment:
Nit: Please remove this line.

Reply:
*to keep the comment about the functionality more generic

Reply:
done